tests/zkevm/test_worst_compute.py

```diff
@@ -741,6 +741,101 @@ def test_worst_unop(
     )
 
 
+@pytest.mark.valid_from("Cancun")
+@pytest.mark.parametrize("key_mut", [True, False])
+@pytest.mark.parametrize("val_mut", [True, False])
```

Comment:
I think it would be helpful to add comments here, since it is not directly clear why `TSTORE` is used in this `TLOAD`  "stress test" when `val_mut = True`. `key_mut` would thus modify the key at the end of each loop (such that the new loop queries a new key) and `val_mut` modifies the value in the transient storage (at least, this is how I understand it from reading the code -> explicitly that this is thus only done at the end of each loop, not at each loop "item")

Reply:
Yes, your interpretation is correct. I'll add comments to help future readers.